plugins/inputs/win_wmi/README.md

```diff
@@ -19,12 +19,27 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
 
 [CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
 
+## Secret-store support
+
+This plugin supports secrets from secret-stores for the `username` and
+`password` option.
+See the [secret-store documentation][SECRETSTORE] for more details on how
+to use them.
+
+[SECRETSTORE]: ../../../docs/CONFIGURATION.md#secret-store-secrets
+
 ## Configuration
 
 ```toml @sample.conf
 # Input plugin to query Windows Management Instrumentation
 # This plugin ONLY supports Windows
 [[inputs.win_wmi]]
+  ## Hostname or IP for remote connections, by default the local machine is queried
```

Comment:
I kind of wonder if this should be more explicit and say "for local machines leave blank"?

Reply:
It's documented on the comments in the sample .conf file isn't it so i would think OK as is.